coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -292,6 +292,46 @@ def matmul(context, node):
     res = mb.matmul(x=inputs[0], y=inputs[1], name=node.name)
     context.add(res)
 
+@register_torch_op
+def einsum(context, node):
+    inputs = _get_inputs(context, node, expected=2)
+    equation = inputs[0].val
+    operands = inputs[1]
+
+    a = operands[0]
+    b = operands[1]
+
+    if equation == "bnqd,bnkd->bnqk":
```

Comment:
It would be good if you create a mapping for the equation?
like mapping different symbol into a fixed dictionary,
so that "bnqd,bnkd->bnqk" and "abcd,abed->anbce" can share the same implementation :)

Reply:
Agreed but I kept the same impl as the TF frontend 🤔 Do you want me to refactor both?

Reply:
We can do a separate enhancement PR for that I think :)

Reply:
CI has an error: https://gitlab.com/zach_nation/coremltools/-/jobs/709610971